.github/workflows/semantic_pr.yaml

```diff
@@ -0,0 +1,35 @@
+name: "Conventional Commit Check"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  build:
+    name: PR title matches the Conventional Commits spec
```

Comment:
Let's close this PR and add this as a step to VeryGoodWorkflows as discussed so that this check is automatically performed for all of our current projects, thanks! 🙏

Reply:
https://github.com/VeryGoodOpenSource/very_good_workflows/pull/45